deploy/buildpacks/publish.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Skaffold Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+PACK_VERSION=0.6.0
```

Comment:
usually, folks end up copying the version string from `pack --version` output
```
 ~/Downloads/pack  --version
v0.6.0 (git sha: 109b629d388cec0ed3836b9fed6717727a9187c1)
tejaldesai-macbookpro2:~ tejaldesai$ 
```
Do you want to add the prefix `v` to the version number?

Reply:
Also follow the same `v0.0.6` version convention for the skaffold gcb image tag?

Reply:
I'm fixing that. Thank you!